known-shortcuts/visualstudio.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:bug: **Bug**
The `AppParams` type is imported but not used in the code. The `app` object should be explicitly typed as `AppParams` to ensure type safety and consistency with the refactoring goals described in the PR.

```suggestion
const app: AppParams = {
```